hosting/setting-up-hostd/download.md

```diff
@@ -0,0 +1,76 @@
+---
+description: Setup a new host on Windows
+layout:
+  title:
+    visible: true
+  description:
+    visible: true
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+
+# Desktop App for Windows: `hostd`
+
+The `hostd` desktop app provides a user-friendly web UI to start storing data on the Sia network. By the end of this guide, you will:
+
+* Have installed `hostd` desktop application.
+* Created a `hostd` wallet
+
+## Pre-requisites
+
+To run `hostd` on Windows, your system should meet the following specifications:
+
+* **Network Access:** `hostd` requires a stable internet connection and open network access to provide storage on the Sia network.
+
+* **System Updates:** Keep Windows updated to get the latest security fixes and improvements.
+
+* **System Requirements:** Quad-core processor, 8GB RAM, a minimum of 256GB SDD for consensus data and 4TB HDD for stored data.
+
+{% hint style="warning" %}
+Storage providing is a commitment that requires technical knowledge, and your machine must meet Sia’s minimum requirements to avoid lost contracts or Siacoin penalties.
+{% endhint %}
+
+## Download
+
+After downloading the `hostd` desktop application:
+
+1. Locate the `.exe` in your **Downloads** folder. We recommend moving it to a convenient location.
```

Comment:
Doesn't it install to a default location?

Reply:
Yeah, sure, but that's the *installer*. Not the actual application. I think it's even called "Setup" when you download it. You double click the installer then it installs to a default location. The actual app is in the start menu/applications folder like every other installed program. You can delete the thing from the downloads folder after it's installed.


https://github.com/user-attachments/assets/6406bb12-e901-4809-92e8-88f7934ac730